src-plugins/vtkDataMesh/readers/vtkDataMeshReader.cpp

```diff
@@ -89,28 +90,124 @@ bool vtkDataMeshReader::read(const QString& path) {
             return false;
         }
 
-        try
-        {
-            dataSet->Read(path.toLocal8Bit().constData());
```

Comment:
I am just reading `vtkMetaSurfaceMesh::Read` method, it manages .vtk, .vtp, .obj, .mesh, .fib.